README.md

```diff
@@ -19,7 +19,9 @@ $ make unit
 $ make integration
 ```
 
-The integration test client points to a local Docker container at port 2525, with the additional ports 8080-8083 exposed for communication with test imposters. Currently tested against a mountebank v1.16.0 instance.
+The integration test client expects a version of mountebank running at `localhost:2525`, with the additional ports 8080-8081 exposed for test imposters.
```

Comment:
Expecting users to have specific software running at specific ports is asking too much IMHO. Maybe running the integration tests in an specific environment build just for that would be a nice idea (docker maybe) to avoid people form having to setup their environment the way the lib wants.

Reply:
See my comment in the Makefile diff and/or commit 335c505.